engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-ostree-branches.html

```diff
@@ -0,0 +1,101 @@
+<span page-title ng-model="repository">{{ 'Manage Branches for Repository:' | translate }} {{ repository.name }}</span>
+
+<a ui-sref="products.details.repositories.info({productId: product.id, repositoryId: repository.id})">
+  <i class="fa fa-angle-double-left"></i>
+  {{ "Back to Repository Details" | translate }}
+</a>
+
+<div>
+  <div data-block="header">
+    <h3 translate>Branches in {{ repository.name }}</h3>
+  </div>
+
+  <div data-block="actions">
+    <div bst-modal="removeBranches()">
+      <div data-block="modal-header"
+           translate-n="getSelectedBranches().length"
+           translate-plural="Remove {{ getSelectedBranches().length }} Branches?">
+        Remove {{ getSelectedBranches().length  }} Branches?
+      </div>
+      <div data-block="modal-body"
+           translate-n="getSelectedBranches().length"
+           translate-plural="Are you sure you want to remove the {{ getSelectedBranches().length }} Branches selected?">
+        Are you sure you want to remove the {{ getSelectedBranches().length }} Branches selected?
+      </div>
+    </div>
+    <div class = "fr">
+      <button class="btn btn-default"
+              ng-click="openModal()"
+              ng-hide="denied('edit_products')"
+              ng-disabled="getSelectedBranches().length === 0">
+        <!-- <i class="fa fa-trash-o" ng-hide="detailsTable.working"></i> -->
+        <span translate>Remove Branches</span>
+      </button>
+    </div>
+  </div>
+
+  <div data-block="table">
+    <table class="table table-striped table-bordered" >
+
+      <thead>
+        <tr>
+          <th class="row-select"><input type="checkbox"
+                                ng-model="allSelected"
```

Comment:
Where is allSelected defined? Generally it is a bad idea (and can cause weird behaviors) to bind ng-model to a primitive (in this case a boolean value) and not an object.